hawkular-inventory-api/src/main/java/org/hawkular/inventory/api/Inventory.java

```diff
@@ -101,6 +110,91 @@
  * @since 0.0.1
  */
 public interface Inventory extends AutoCloseable, Tenants.Container<Tenants.ReadWrite> {
+    static final class InventotyUtils {
+        private static final Map<String, Class<? extends Entity<?, ?>>>entityTypes;
+        static {
+            Map<String, Class<? extends Entity<?, ?>>> m = Arrays.asList(
+                    Tenant.class,
+                    Environment.class,
+                    Feed.class,
+                    ResourceType.class,
+                    MetricType.class,
+                    Resource.class,
+                    Metric.class,
+                    OperationType.class,
+                    MetadataPack.class)
+            .stream().collect(Collectors.toMap(Class::getName, Function.identity()));
+            entityTypes = Collections.unmodifiableMap(m);
+        }
+        static String shortenIfNeeded(CanonicalPath path) {
```

Comment:
There is a reference to `getStableId()` outside the security package: https://github.com/hawkular/hawkular-inventory/pull/259/files#diff-0646e1614dd0f2d2c68141bcd5bc6fdaR107 which is why I moved it here where both REST API and Security implementations can depend on it.